dummyserver/certs/cacert.pem

```diff
@@ -1,23 +1,23 @@
 -----BEGIN CERTIFICATE-----
 MIIDzDCCAzWgAwIBAgIJALPrscov4b/jMA0GCSqGSIb3DQEBBQUAMIGBMQswCQYD
-VQQGEwJGSTEOMAwGA1UECBMFZHVtbXkxDjAMBgNVBAcTBWR1bW15MQ4wDAYDVQQK
-EwVkdW1teTEOMAwGA1UECxMFZHVtbXkxETAPBgNVBAMTCFNuYWtlT2lsMR8wHQYJ
+VQQGEwJGSTEOMAwGA1UECAwFZHVtbXkxDjAMBgNVBAcMBWR1bW15MQ4wDAYDVQQK
```

Comment:
Literally nothing.

I had a real saga where macOS repeatedly refused to allow this as a CA, refusing to trust this certificate unless explicitly instructed to (which is not very helpful when you have a programmatic API). In the end I rebuilt the certificate to have identical data using cryptography (I'm serious, run `openssl x509` on both the original and new cert and you'll see they are the same except for their signature)` and found that it started to work. Presumably there was some garbage in there somewhere that macOS, in its strictness, didnt' like.